crypto/ec/curve448/curve448_tables.c

```diff
@@ -16,325 +16,325 @@
 static const curve448_precomputed_s curve448_precomputed_base_table = {
     {
         {{
-            {FIELD_LITERAL(0x00cc3b062366f4cc,0x003d6e34e314aa3c,0x00d51c0a7521774d,0x0094e060eec6ab8b,0x00d21291b4d80082,0x00befed12b55ef1e,0x00c3dd2df5c94518,0x00e0a7b112b8d4e6)},
-            {FIELD_LITERAL(0x0019eb5608d8723a,0x00d1bab52fb3aedb,0x00270a7311ebc90c,0x0037c12b91be7f13,0x005be16cd8b5c704,0x003e181acda888e1,0x00bc1f00fc3fc6d0,0x00d3839bfa319e20)},
-            {FIELD_LITERAL(0x003caeb88611909f,0x00ea8b378c4df3d4,0x00b3295b95a5a19a,0x00a65f97514bdfb5,0x00b39efba743cab1,0x0016ba98b862fd2d,0x0001508812ee71d7,0x000a75740eea114a)},
+            {FIELD_LITERAL(0x00cc3b062366f4ccuLL,0x003d6e34e314aa3cuLL,0x00d51c0a7521774duLL,0x0094e060eec6ab8buLL,0x00d21291b4d80082uLL,0x00befed12b55ef1euLL,0x00c3dd2df5c94518uLL,0x00e0a7b112b8d4e6uLL)},
+            {FIELD_LITERAL(0x0019eb5608d8723auLL,0x00d1bab52fb3aedbuLL,0x00270a7311ebc90cuLL,0x0037c12b91be7f13uLL,0x005be16cd8b5c704uLL,0x003e181acda888e1uLL,0x00bc1f00fc3fc6d0uLL,0x00d3839bfa319e20uLL)},
+            {FIELD_LITERAL(0x003caeb88611909fuLL,0x00ea8b378c4df3d4uLL,0x00b3295b95a5a19auLL,0x00a65f97514bdfb5uLL,0x00b39efba743cab1uLL,0x0016ba98b862fd2duLL,0x0001508812ee71d7uLL,0x000a75740eea114auLL)},
```

Comment:
the line length is already too long, but it is difficult to see, if the values are changed or not.
that should probably done in the FIELD_LITERAL macro.

Reply:
Or break lines:

``` C
            {FIELD_LITERAL(0x003caeb88611909fuLL,0x00ea8b378c4df3d4uLL,
                           0x00b3295b95a5a19auLL,0x00a65f97514bdfb5uLL,
                           0x00b39efba743cab1uLL,0x0016ba98b862fd2duLL,
                           0x0001508812ee71d7uLL,0x000a75740eea114auLL)},
```

Reply:
Not sure that it's such a good idea to hide the size suffix like that... that makes for harder to read code.

Reply:
I did think of doing that, and then thought it was probably not a good idea. I know on a previous commit in another PR someone didnt like using my use of the string tokeniser.

Reply:
Changed to use tokeniser